### ISSUE TITLE
sequential clock trigger bug

### ISSUE THREAD

**Issue by hjoliver** [cylc/cylc-flow#6204]
Only one task can trigger off of a sequential clock trigger, after the first cycle.

This only affects clock triggers, not other xtriggers.

Example: after the first cycle, only a, p, and q will spawn ahead and run:

```
[scheduler]
    allow implicit tasks = True
[scheduling]
    initial cycle point = 2000
    [[xtriggers]]
        x = wall_clock(sequential=True)
        y = xrandom(50, _=%(point)s, sequential=True)
    [[graph]]
        PT1M = """
            @x => a & b
            @y => p & q
        """
```
Setting `sequential=False` restores correct (but non-sequential) behaviour.

**Comment by dwsutherland** [comment]
Found the problem: If a wall_clock xtrigger had already been satisfied, the task ids weren't added to the spawn next set.

**Comment by oliver-sanders** [comment]
Closed by #6206
